reference/sql/data-types/date-and-time.md

```diff
@@ -76,6 +78,8 @@ Different types of zero value are shown in the following table:
 
 Invalid `DATE`, `DATETIME`, `TIMESTAMP` values are automatically converted to the corresponding type of zero value ( '0000-00-00' or '0000-00-00 00:00:00' ) if the SQL mode permits such usage.
 
+<!-- markdownlint-disable MD001 -->
```

Comment:
@TomShawn Because if the rule is not added, the ci will fail

Reply:
That's because a heading level in this document increments by more than one level at a time, which is allowed by CI. See Line 79 and update accordingly.

Reply:
I was wondering if we can disable this rule at the first place. Because it is strange if `### Automatic initialization and update of TIMESTAMP and DATETIME` is at the same level with `## Supported types`. If disabling this rule is not recommended, I prefer to reorganize the content, for example, move these content (under heading 3) to the end of this doc~

Reply:
Good idea. Please help reorganize the content and refine the heading texts in another PR, thanks~ @toutdesuite